Ambiguous Documentation fix for guvectorize. (#4047)
numba/numba: docs/source/user/vectorize.rst

  array, a scalar (symbolically denoted by the empty tuple ``()``) and
  returns a *n*-element one-dimension array;

> * the list of supported concrete *signatures* are the same as in ``@vectorize``; here

Reviewer: ```suggestion
* the list of supported concrete *signatures* as per ``@vectorize``; here,
```

I think this statement needs to allude to that the provision of concrete signatures is the same as noted for `@vectorize`.

>   in the above example we demonstrate ``int64`` arrays.

Reviewer: ```suggestion
  as in the above example, we demonstrate ``int64`` arrays.
```
